yarn-project/circuit-types/src/logs/function_l2_logs.ts

```diff
@@ -37,6 +37,16 @@ export abstract class FunctionL2Logs<TLog extends UnencryptedL2Log | EncryptedL2
     return this.logs.reduce((acc, log) => acc + log.length + 4, 0) + 4;
   }
 
+  /**
+   * Get the total length of all chargable data (raw log data + 4 for each log)
+   * TODO: Rename this? getChargableLength? getDALength?
```

Comment:
Unsure on the name `getKernelLength` - this method returns the lengths we expect from the kernels. So this doesn't include the +4 bytes added to the serialised logs for each function call and each blob of logs.